sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerClient.cs

```diff
@@ -78,11 +78,19 @@ internal ConfidentialLedgerClient(Uri ledgerEndpoint, TokenCredential credential
                     Array.Empty<HttpPipelinePolicy>() :
                     new HttpPipelinePolicy[] { new BearerTokenAuthenticationPolicy(_tokenCredential, AuthorizationScopes) },
                 transportOptions,
-                new ResponseClassifier());
+                new ConfidentialLedgerResponseClassifier());
```

Comment:
Does this client have other uses besides being the underlying client for the PostLedgerEntryOperation? If so, do we want this new behavior for all scenarios, or does it need to be scoped to the specific scenario?

Reply:
Yes this client does have other underlying uses including Getting Entries, User Data and other information. We observed this bug in the `PostLedgerEntryOperationAsync` method but it may occur in various other methods where our client is reaching out to our service and reading from a different node that it has written to. I've updated the changelog to mention that we are bringing this change to all scenarios.